qutebrowser/utils/utils.py

```diff
@@ -216,13 +220,12 @@ def read_file(filename: str, binary: bool = False) -> Any:
             with open(fn, 'r', encoding='utf-8') as f:
                 return f.read()
     else:
-        data = pkg_resources.resource_string(
-            qutebrowser.__name__, filename)
+        p = importlib_resources.files(qutebrowser) / filename
```

Comment:
Conveniently for line length, importlib_resources supports passing a module, not just a str.